ui/src/environments/environment.local.ts

```diff
@@ -25,5 +25,5 @@ export const environment: Environment = {
   enablePublishedWorkspaces: false,
   enableProfileCapsFeatures: true,
   enableNewConceptTabs: true,
-  enableSignedInFooter: true
+  enableSignedInFooter: false
```

Comment:
we are hiding it from local and test?

Reply:
I am hiding it from local and test because we are doing UX verification on those environments leading up to launch, and while we are doing that testing I wanted to make sure that the footer wasn't there, since we aren't going to have it for launch.